apps/parsley/src/pages/LogDrop.tsx

```diff
@@ -1,10 +1,11 @@
+import { usePageTitle } from "@evg-ui/lib/hooks/usePageTitle";
 import LogWindow from "components/LogWindow";
 import { useLogContext } from "context/LogContext";
 import FileDropper from "./LogDrop/FileDropper";
 
 const LogDrop = () => {
-  const { hasLogs } = useLogContext();
-
+  const { hasLogs, logMetadata } = useLogContext();
+  usePageTitle(`Parsley | ${hasLogs ? logMetadata?.fileName : "Upload logs"}`);
```

Comment:
I would consider putting "Parsley" at the end of the string, otherwise the contextual information is liable to get cut off. The favicon does a good job of indicating the tab is Parsley.